### ISSUE TITLE
'ViewPropTypes' is not exported from 'react-native-web/dist/index

### ISSUE THREAD

**Issue by vitalyiegorov** [FaridSafi/react-native-gifted-chat#1683]
#### Issue Description
`/node_modules/react-native-gifted-chat/lib/MessageImage.js
Attempted import error: 'ViewPropTypes' is not exported from 'react-native-web/dist/index'.`

#### Steps to Reproduce / Code Snippets
Upgrade react-native-web to 0.12.2

#### Expected Results
Everything works fine

#### Additional Information
Due to new release of RNW:
https://github.com/necolas/react-native-web/issues/1537

* Nodejs version: 10.15.3
* React version: 16.13.0
* React Native version: 0.12.2
* react-native-gifted-chat version: 0.13.0
* Platform(s) (iOS, Android, or both?): both
* TypeScript version: 3.8.3

**Comment by littlehome-eugene** [comment]
./node_modules/react-native-ratings/src/SwipeRating.js
Attempted import error: 'ViewPropTypes' is not exported from 'react-native'.

similar issue

**Comment by jayb967** [comment]
I am having the same exact issue, any one have a solution?

**Comment by rachedwael** [comment]
I am having the same exact issue

**Comment by vitalyiegorov** [comment]
@FaridSafi Can we contribute and exclude PropTypes from components, or any other way?

**Comment by xkungfu** [comment]
hi, i got issue like yours.
did you resolve it?
thanks!

and I created a new issue: https://github.com/FaridSafi/react-native-gifted-chat/issues/1963

**Comment by jackphuongvu** [comment]
Same issue here guys

**Comment by BrianLi101** [comment]
**tldr**: use our fork of [react-native-gifted-chat](https://github.com/gtechnologies/react-native-gifted-chat) to run it on web

If you are using version 0.10.0 of react-native-gifted-chat, you will have to use a version of [react-native-web](https://github.com/necolas/react-native-web) that is earlier than 0.12.0 because the deprecated ViewPropTypes to stay in line with changes to React Native. I wouldn't recommend this because quite a significant number of changes have happened since then.

If you try to use version 0.16.3 of [react-native-gifted-chat](https://github.com/FaridSafi/react-native-gifted-chat), you will likely run into an issue with `TypingAnimation` imported from `react-native-typing-animation` causing the following error: "Support for the experimental syntax 'jsx' isn't currently enabled".

My team has forked react-native-gifted-chat and built a patch solution for the time being: https://github.com/gtechnologies/react-native-gifted-chat.

It also has a much more thorough explanation of the problems we encountered on web. Hopefully this helps!
